client-app/app/components/page-nav.js

```diff
@@ -0,0 +1,45 @@
+import Component from "@ember/component";
+import { computed, observer } from "@ember/object";
+import { later } from "@ember/runloop";
+
+export default Component.extend({
+  tagName: "div",
+  classNames: ["nav-controls"],
+  classNameBindings: ["extraClasses"],
+
+  didReceiveAttrs() {
+    this._super(...arguments);
+    later(this, () => {
+      this.navigate(this.position);
+    });
+  },
+
+  resetPosition: observer("resetPositionOnChange", function() {
```

Comment:
Can't we get read of this observer?

Reply:
Yes, I've removed it. When I picked up this PR again I refactored it to make it follow the DDAU practice as much as possible, and that made removing this observer super easy since you know only one place can modify an attribute and you know what that place is.